<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1122</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>VTree</string>
  </property>
  <property name="toolTipDuration">
   <number>-7</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #E6E6EA;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical {
                        width: 45px;
                        margin: 45px 0 45px 0;
                      }

                      QScrollBar::handle:vertical {
                        min-height: 10px;
                      }

                      QScrollBar::add-line:vertical {
                        background: none;
                        height: 45px;
                        subcontrol-position: bottom;
                        subcontrol-origin: margin;
                      }

                      QScrollBar::sub-line:vertical {
                        background: none;
                        height: 45px;
                        subcontrol-position: top;
                        subcontrol-origin: margin;
                      }

                      QScrollBar::up-arrow:vertical { 
                        image:url('./icons/up_48.png'); 
                        height: 40px; 
                        width: 40px 
                      }

                      QScrollBar::down-arrow:vertical {
                        image:url('./icons/down_48.png'); 
                        height: 40px; 
                        width: 40px                              
                      }</string>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>1771</x>
      <y>116</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #60a917;
padding: 10px;
border: 0;
</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="addBox">
    <property name="geometry">
     <rect>
      <x>1661</x>
      <y>75</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	background-color: white;
	border: 0;
}

QSpinBox::up-button  {
	border: 0px;
	height: 0;
	width: 0;
}

QSpinBox::down-button  {
	border: 0px;
	height: 0;
	width: 0;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-10000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseButton">
    <property name="geometry">
     <rect>
      <x>1661</x>
      <y>116</y>
      <width>111</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
padding: 10px;
border: 0;
</string>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="orderAdd">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>280</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #60a917;
padding: 10px;
border: 0;
</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="orderErase">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>280</y>
      <width>111</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey;
padding: 10px;
border: 0;
</string>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="leftValue">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>241</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	background-color: white;
	border: 0;
}

QSpinBox::up-button  {
	border: 0px;
	height: 0;
	width: 0;
}

QSpinBox::down-button  {
	border: 0px;
	height: 0;
	width: 0;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>-10000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="rightValue">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>241</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
	background-color: white;
	border: 0;
}

QSpinBox::up-button  {
	border: 0px;
	height: 0;
	width: 0;
}

QSpinBox::down-button  {
	border: 0px;
	height: 0;
	width: 0;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1661</x>
      <y>35</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>11</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Одичночные операции</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1658</x>
      <y>200</y>
      <width>220</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Массовые операции</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>440</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border:                1px solid #ccc ;
    border-radius:         4px;
	background-color: white;
	text-align: center;
}
QComboBox::drop-down 
{
    border: 0px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>AVL</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::NoInsert</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="minimumContentsLength">
     <number>3</number>
    </property>
    <property name="duplicatesEnabled">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>AVL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Красно-черное</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Splay</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>410</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Вид дерева</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="orderErase_2">
    <property name="geometry">
     <rect>
      <x>1599</x>
      <y>1023</y>
      <width>321</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andale Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9E9E9E;
padding: 10px;
border: 0;
</string>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
